--- libfslcodec-3.0.1.orig/Makefile.in	2012-09-07 13:49:57.000000000 +0200
+++ libfslcodec-3.0.1/Makefile.in	2013-07-19 13:44:29.351294501 +0200
@@ -1214,7 +1214,7 @@
     $(fslpngdecinclude)           \
     $(fslgifdecinclude)
 
-libfslaudiocodeclibdir = $(libdir)/imx-mm/audio-codec
+libfslaudiocodeclibdir = $(libdir)
 libfslaudiocodeclib_DATA = \
     $(fslpeqlib) 	   \
     $(fslsrclib) 	   \
@@ -1232,13 +1232,13 @@
     $(fslamrnblib)     \
     $(fslamrwblib)
 
-libfslaudiowraplibdir = $(libdir)/imx-mm/audio-codec/wrap
+libfslaudiowraplibdir = $(libdir)
 libfslaudiowraplib_DATA = \
     $(fslaacdecwraplib)   \
     $(fslmp3decwraplib)   \
     $(fslvorbisdecwraplib)
 
-libfslvideocodeclibdir = $(libdir)/imx-mm/video-codec
+libfslvideocodeclibdir = $(libdir)
 libfslvideocodeclib_DATA = \
     $(fsldeinterlacelib)   \
     $(fslh264declib)       \
