diff -Naur imx-gst1-plugin-rel_imx_4.9.88_2.0.0_ga-orig/configure.ac imx-gst1-plugin-rel_imx_4.9.88_2.0.0_ga-new/configure.ac
--- imx-gst1-plugin-rel_imx_4.9.88_2.0.0_ga-orig/configure.ac	2018-03-20 09:56:18.000000000 +0100
+++ imx-gst1-plugin-rel_imx_4.9.88_2.0.0_ga-new/configure.ac	2019-04-02 11:26:03.108816582 +0200
@@ -32,7 +32,7 @@
 AC_DEFUN([FIND_PARSER_SONAME],
 [
     AC_MSG_CHECKING([for lib_$1_parser_$2_elinux.3.0.so soname])
-    path=`$PKG_CONFIG --variable=libdir imx-parser`
+    path=`PKG_CONFIG_SYSROOT_DIR="" $PKG_CONFIG --variable=libdir imx-parser`
     SONAME=$( $CC -print-file-name=$PKG_CONFIG_SYSROOT_DIR$path/lib_$1_parser_$2_elinux.so | \
 	    while read output; do objdump -p $output | \
 	    grep "SONAME" | \
@@ -51,7 +51,7 @@
 AC_DEFUN([FIND_ACODEC_SONAME],
 [
     AC_MSG_CHECKING([for lib_$1_wrap_$2_elinux.so soname])
-    path=`$PKG_CONFIG --variable=libdir libfslaudiocodec`
+    path=`PKG_CONFIG_SYSROOT_DIR="" $PKG_CONFIG --variable=libdir libfslaudiocodec`
     SONAME=$( $CC -print-file-name=$PKG_CONFIG_SYSROOT_DIR$path/wrap/lib_$1_wrap_$2_elinux.so | \
 	    while read output; do objdump -p $output | \
 	    grep "SONAME" | \
