--- a/plat/imx/imx8m/ddr/dram.c	2020-09-18 09:58:51.485197683 +0000
+++ b/plat/imx/imx8m/ddr/dram.c	2020-09-18 10:00:34.710801397 +0000
@@ -219,7 +219,8 @@
 		dcsw_op_all(DCCSW);
 		lpddr4_swffc(&dram_info, dev_fsp, 0x0);
 		dev_fsp = (~dev_fsp) & 0x1;
-	} else if (current_fsp != 0x0) {
+	/* Does not work with DDR3L so we have to disable it for DDR3L for now */
+	} else if (dram_info.dram_type == DDRC_DDR4 && current_fsp != 0x0) {
 		/* flush the L1/L2 cache */
 		dcsw_op_all(DCCSW);
 		ddr4_swffc(&dram_info, 0x0);
@@ -307,7 +308,8 @@
 		if (dram_info.dram_type == DDRC_LPDDR4) {
 			lpddr4_swffc(&dram_info, dev_fsp, fsp_index);
 			dev_fsp = (~dev_fsp) & 0x1;
-		} else {
+		/* Does not work with DDR3L so we have to disable it for DDR3L for now */
+		} else if (dram_info.dram_type == DDRC_DDR4){
 			ddr4_swffc(&dram_info, fsp_index);
 		}
