diff -Naur imx-gst1-plugin-4.0.9-orig/configure.ac imx-gst1-plugin-4.0.9-new/configure.ac
--- imx-gst1-plugin-4.0.9-orig/configure.ac	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/configure.ac	2016-06-13 17:17:14.000000000 +0200
@@ -32,7 +32,7 @@
 AC_DEFUN([FIND_PARSER_SONAME],
 [
     AC_MSG_CHECKING([for lib_$1_parser_$2_elinux.3.0.so soname])
-    path=`$PKG_CONFIG --variable=libdir libfslparser`
+    path=`PKG_CONFIG_SYSROOT_DIR="" $PKG_CONFIG --variable=libdir libfslparser`
     SONAME=$( $CC -print-file-name=$PKG_CONFIG_SYSROOT_DIR$path/lib_$1_parser_$2_elinux.so | \
 	    while read output; do objdump -p $output | \
 	    grep "SONAME" | \
@@ -51,7 +51,7 @@
 AC_DEFUN([FIND_ACODEC_SONAME],
 [
     AC_MSG_CHECKING([for lib_$1_wrap_$2_elinux.so soname])
-    path=`$PKG_CONFIG --variable=libdir libfslaudiocodec`
+    path=`PKG_CONFIG_SYSROOT_DIR="" $PKG_CONFIG --variable=libdir libfslaudiocodec`
     SONAME=$( $CC -print-file-name=$PKG_CONFIG_SYSROOT_DIR$path/wrap/lib_$1_wrap_$2_elinux.so | \
 	    while read output; do objdump -p $output | \
 	    grep "SONAME" | \
