diff -Naur imx-gst1-plugin-4.0.9-orig/libs/allocator/gstallocatorphymem.c imx-gst1-plugin-4.0.9-new/libs/allocator/gstallocatorphymem.c
--- imx-gst1-plugin-4.0.9-orig/libs/allocator/gstallocatorphymem.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/libs/allocator/gstallocatorphymem.c	2016-06-15 15:35:45.000000000 +0200
@@ -18,6 +18,7 @@
  */
 
 #include <stdio.h>
+#include <string.h>
 #include "gstallocatorphymem.h"
 
 typedef struct {
diff -Naur imx-gst1-plugin-4.0.9-orig/libs/device-2d/imx_2d_device_ipu.c imx-gst1-plugin-4.0.9-new/libs/device-2d/imx_2d_device_ipu.c
--- imx-gst1-plugin-4.0.9-orig/libs/device-2d/imx_2d_device_ipu.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/libs/device-2d/imx_2d_device_ipu.c	2016-06-15 15:33:33.000000000 +0200
@@ -18,6 +18,7 @@
  */
 
 #include <fcntl.h>
+#include <unistd.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <linux/ipu.h>
diff -Naur imx-gst1-plugin-4.0.9-orig/libs/gstimxcommon.h imx-gst1-plugin-4.0.9-new/libs/gstimxcommon.h
--- imx-gst1-plugin-4.0.9-orig/libs/gstimxcommon.h	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/libs/gstimxcommon.h	2016-06-13 18:06:14.000000000 +0200
@@ -21,6 +21,7 @@
 #define __IMX_COMMON_H__
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/utsname.h>
 #include <gst/gst.h>
 #include <string.h>
diff -Naur imx-gst1-plugin-4.0.9-orig/libs/gstsutils/gstsutils.c imx-gst1-plugin-4.0.9-new/libs/gstsutils/gstsutils.c
--- imx-gst1-plugin-4.0.9-orig/libs/gstsutils/gstsutils.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/libs/gstsutils/gstsutils.c	2016-06-13 18:06:45.000000000 +0200
@@ -32,6 +32,7 @@
 *
 */
 
+#include <string.h>
 #include "gstsutils.h"
 
 typedef struct _GstsutilsData
diff -Naur imx-gst1-plugin-4.0.9-orig/libs/v4l2_core/gstimxv4l2.c imx-gst1-plugin-4.0.9-new/libs/v4l2_core/gstimxv4l2.c
--- imx-gst1-plugin-4.0.9-orig/libs/v4l2_core/gstimxv4l2.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/libs/v4l2_core/gstimxv4l2.c	2016-06-13 18:07:32.000000000 +0200
@@ -28,6 +28,7 @@
 #include <linux/mxcfb.h>
 #include <linux/ipu.h>
 #include <linux/mxc_v4l2.h>
+#include <linux/videodev2.h>
 #include <linux/version.h>
 #include <gst/video/gstvideosink.h>
 
diff -Naur imx-gst1-plugin-4.0.9-orig/libs/video-overlay/gstimxvideooverlay.c imx-gst1-plugin-4.0.9-new/libs/video-overlay/gstimxvideooverlay.c
--- imx-gst1-plugin-4.0.9-orig/libs/video-overlay/gstimxvideooverlay.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/libs/video-overlay/gstimxvideooverlay.c	2016-06-15 15:35:02.000000000 +0200
@@ -18,6 +18,7 @@
  */
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include "gstimxvideooverlay.h"
 
diff -Naur imx-gst1-plugin-4.0.9-orig/plugins/aiurdemux/aiurcontent.c imx-gst1-plugin-4.0.9-new/plugins/aiurdemux/aiurcontent.c
--- imx-gst1-plugin-4.0.9-orig/plugins/aiurdemux/aiurcontent.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/plugins/aiurdemux/aiurcontent.c	2016-06-15 15:31:42.000000000 +0200
@@ -18,6 +18,8 @@
  */
 
 #include <string.h>
+#include <stdlib.h>
+#include <sys/stat.h>
 #include "aiurcontent.h"
 
 GST_DEBUG_CATEGORY_EXTERN (aiurdemux_debug);
diff -Naur imx-gst1-plugin-4.0.9-orig/plugins/aiurdemux/aiurtypefind.c imx-gst1-plugin-4.0.9-new/plugins/aiurdemux/aiurtypefind.c
--- imx-gst1-plugin-4.0.9-orig/plugins/aiurdemux/aiurtypefind.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/plugins/aiurdemux/aiurtypefind.c	2016-06-15 15:32:37.000000000 +0200
@@ -34,6 +34,7 @@
   *
   * Portability:    This code is written for Linux OS and Gstreamer
   */
+#include <string.h>
 #include <gst/gst.h>
 
 typedef void (*AiurTypeFindFunc) (GstTypeFind *, gpointer);
diff -Naur imx-gst1-plugin-4.0.9-orig/plugins/vpu/gstvpu.c imx-gst1-plugin-4.0.9-new/plugins/vpu/gstvpu.c
--- imx-gst1-plugin-4.0.9-orig/plugins/vpu/gstvpu.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/plugins/vpu/gstvpu.c	2016-06-15 15:29:14.000000000 +0200
@@ -17,6 +17,7 @@
  * Boston, MA 02111-1307, USA.
  */
 
+#include <string.h>
 #include "gstvpuenc.h"
 
 gint
diff -Naur imx-gst1-plugin-4.0.9-orig/tools/gplay/playengine.c imx-gst1-plugin-4.0.9-new/tools/gplay/playengine.c
--- imx-gst1-plugin-4.0.9-orig/tools/gplay/playengine.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/tools/gplay/playengine.c	2016-06-15 15:26:12.000000000 +0200
@@ -13,6 +13,8 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include <unistd.h>
+#include <sys/ioctl.h>
 #include <gst/gst.h>
 #include <gst/video/videooverlay.h>
 #include <gst/app/gstappsink.h>
diff -Naur imx-gst1-plugin-4.0.9-orig/tools/gplay/playlist.c imx-gst1-plugin-4.0.9-new/tools/gplay/playlist.c
--- imx-gst1-plugin-4.0.9-orig/tools/gplay/playlist.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/tools/gplay/playlist.c	2016-06-15 15:28:02.000000000 +0200
@@ -41,6 +41,8 @@
  * =====================================================================================
  */
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
 #include "playlist.h"
 
 #define MEM_ALLOC(size) malloc((size))
diff -Naur imx-gst1-plugin-4.0.9-orig/tools/grecorder/grecorder.c imx-gst1-plugin-4.0.9-new/tools/grecorder/grecorder.c
--- imx-gst1-plugin-4.0.9-orig/tools/grecorder/grecorder.c	2016-03-10 16:08:42.000000000 +0100
+++ imx-gst1-plugin-4.0.9-new/tools/grecorder/grecorder.c	2016-06-15 15:23:15.000000000 +0200
@@ -20,6 +20,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 #include <semaphore.h>
 #include <pthread.h>
 #include <signal.h>
