--- a/Makefile_G1G2
+++ b/Makefile_G1G2
@@ -10,7 +10,11 @@
 
 INCLUDE_HEADERS += -I$(SOURCE_ROOT)/linux/memalloc
 #INCLUDE_HEADERS += -I$(SOURCE_ROOT)/linux/ldriver
-INCLUDE_HEADERS += -I$(SDKTARGETSYSROOT)/usr/include/imx
+#INCLUDE_HEADERS += -I$(SDKTARGETSYSROOT)/usr/include/imx
+INCLUDE_HEADERS += -I$(LINUX_KERNEL_ROOT)/include/uapi -I$(LINUX_KERNEL_ROOT)/include
+INCLUDE_HEADERS += -I$(LINUX_KERNEL_ROOT)/include/generated/uapi
+# ION header location
+INCLUDE_HEADERS += -I$(LINUX_KERNEL_ROOT)/drivers/staging/android/uapi
 
 CFLAGS += -DDEC_MODULE_PATH=\"/dev/mxc_hantro\" -DUSE_FAKE_RFC_TABLE -DFIFO_DATATYPE=void* -DNDEBUG -DDOWN_SCALER \
            -DUSE_EXTERNAL_BUFFER -DUSE_FAST_EC -DUSE_VP9_EC -DGET_FREE_BUFFER_NON_BLOCK \

