diff -Naur xorg-server-1.20.5-old/glamor/Makefile.am xorg-server-1.20.5-new/glamor/Makefile.am
--- xorg-server-1.20.5-old/glamor/Makefile.am	2019-05-30 20:27:34.000000000 +0200
+++ xorg-server-1.20.5-new/glamor/Makefile.am	2022-10-14 14:48:26.394558884 +0200
@@ -2,7 +2,7 @@
 
 libglamor_la_LIBADD = $(GLAMOR_LIBS)
 
-AM_CFLAGS = $(CWARNFLAGS) $(DIX_CFLAGS) $(GLAMOR_CFLAGS)
+AM_CFLAGS = $(CWARNFLAGS) $(DIX_CFLAGS) $(GLAMOR_CFLAGS) $(GBM_CFLAGS)
 
 libglamor_la_SOURCES = \
 	glamor.c \
